src/extractors/custom/www.rbbtoday.com/index.js

```diff
@@ -0,0 +1,34 @@
+export const WwwRbbtodayComExtractor = {
+  domain: 'www.rbbtoday.com',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  author: {
+    selectors: ['.writer.writer-name'],
+  },
+
+  date_published: {
+    selectors: [['header time', 'datetime']],
+  },
+
+  dek: null,
```

Comment:
The `.arti-summary` element actually serves as a `dek` for articles on this website. In most cases, it appears to be excerpt from the first paragraph, but the presence of this element on the pages indicates that this text could sometimes be customized.

Reply:
Thank you.
I added the `.arti-summary` element to the dek.